fsharp-backend/tests/Tests/LibExecution.Tests.fs

```diff
@@ -51,15 +54,40 @@ let setUpWorkers meta workers =
     do! Canvas.saveTLIDs meta oplists
   }
 
+let setupStaticAssets
+  (meta : Canvas.Meta)
+  (deployHashes : List<string>)
```

Comment:
nitpick that we can move elsewhere if better:

built-in types like `list` are generally (at least, by docs I've read, incl. built-in types like `list` are generally (at least, by docs I've read) used in the suffix notation (`x list` rather than `List<x>`)
I'm totally fine getting away from that, but we should mark that in the coding guide.) used in the suffix notation (`x list` rather than `List<x>`), standardize to that, and (as we see it) adjust to use that generally.

There's a LinkedList/Generic.List type that also exists in .NET

upon seeing List<x>, I think many devs brains would think of that mutable array list thing first, esp. if any C# experience